test/integration/unprivileged_newproject_test.go

```diff
@@ -78,6 +78,14 @@ func TestUnprivilegedNewProject(t *testing.T) {
 
 	waitForProject(t, valerieOpenshiftClient, "new-project", 5*time.Second, 10)
 
+	actualProject, err := valerieOpenshiftClient.Projects().Get("new-project")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	if e, a := "valerie", actualProject.Annotations[projectapi.ProjectRequester]; e != a {
+		t.Errorf("expected %v, got %v", e, a)
```

Comment:
Having previously chased these types of error messages, I'd prefer a more verbose message.